Homework Submission from Meisha Hill

meishahill opened the pull request.

meishahill committed:
Cleaned up indentation

wontaeyang commented on my_index.html:
```diff
@@ -0,0 +1,59 @@
+<body>
+		 <div id="top_container"> 
+			<h2> Relaxr </h2>
+			<br>
```
try using padding and margins instead of br tags

wontaeyang commented:
:hammer: one minor issue :D otherwise great!

meishahill committed:
Updated homework submission

Removed <br> and added margin in css

wontaeyang commented:
![image](https://cloud.githubusercontent.com/assets/1811312/10528328/980fcbec-7363-11e5-8805-59dc5de8ce18.png)

something deleted by accident?

wontaeyang commented:
![image](https://cloud.githubusercontent.com/assets/1811312/10528514/ae587cc2-7364-11e5-87ff-b64accfb582a.png)

if you hardcode height, overflowing elements start to spill out

meishahill committed:
Updates to index in CSS

-Put footer back in
-Removed height on middle container but I’m still unsure how I can
maintain the layout if I remove height from background images

meishahill committed:
Updates to css

meishahill commented:
@wontaeyang Homework has been updated!
